config/routechangerconfig.go

```diff
@@ -6,17 +6,25 @@ import (
 	"strings"
 )
 
-type routeChangerConfig struct {
+type routeChangerConfigItem struct {
 	Reg  *regexp.Regexp
 	Repl string
 	Sep  string
 }
 
+type routeChangerConfig []routeChangerConfigItem
+
 func (rcc routeChangerConfig) String() string {
-	if rcc.Reg == nil {
-		return ""
+	result := ""
```

Comment:
https://pkg.go.dev/strings#Builder may be used here like so https://go.dev/play/p/d4Ub_GXEmoP